docs/add-your-app.md

```diff
@@ -0,0 +1,138 @@
+# Adding Your Apps
+
```

Comment:
Please also add a disclaimer that we cannot add an app using GUI. Maybe have a Current Limitations section?

Reply:
Added a reference to the fact it can only be done via API in the final section. not having it in UI is not a "limitation" the entire UI piece is a bonus :)

Reply:
The above was what you used the reply feature and looks fine